pkg/cmd/openshift-sdn/cmd.go

```diff
@@ -28,19 +29,23 @@ import (
 	"github.com/openshift/origin/pkg/version"
 )
 
+const openshiftCNIFile string = "80-openshift-network.conf"
```

Comment:
You left the definition of this in `pkg/network/node/node.go` but I think it's unused now? If it's needed in both places then it should only be defined in one of them and then imported in the other